### PR TITLE
Trouble shooting

### PR DESCRIPTION
**Pull request by F-Sossi**
This is much nicer don't think I have much more to improve

### CODE REVIEW

File: src/sGLOH2.cpp [hunk 1]
```diff
@@ -207,26 +207,58 @@ cv::Mat sGLOH2::compute_sGLOH_single(const cv::Mat& patch) {
  * @param patch The image patch to be processed. It must be a single-channel matrix of type CV_8U.
  * @return The sGLOH2 descriptor for the patch as a single-row matrix of type CV_32F.
  */
+//cv::Mat sGLOH2::compute_sGLOH(const cv::Mat& patch) {
```
Comment (jboyer4) [inline src/sGLOH2.cpp:266]: Do we still need this old code?

File: src/sGLOH2.cpp [hunk 2]
```diff
@@ -32,7 +32,7 @@
 constexpr int PATCH_SIZE = 41;
 constexpr int N = 2;
 constexpr int M = 4;
-constexpr int Q = 8;
+constexpr int Q = 4;
```
Comment (jboyer4) [inline src/sGLOH2.cpp:35]: I understand M, N, and PATCHSIZE - what is Q doing here though?

Comment (jboyer4) [reply src/sGLOH2.cpp:35]: I know we talked about it the other day, but I think I'm still not quite on the same page on that one

**Review by jboyer4 — APPROVED** [review]

File: src/sGLOH2.cpp (hunk 2 above)
Comment (F-Sossi) [reply src/sGLOH2.cpp:35]: I had q for the blocks and it has slowly turned into M going to remove this just wanted your testing to be on the current version. Should just be clean up and tweaking from now till turn in unless I find another part that really bothers me.

Comment (jboyer4) [reply src/sGLOH2.cpp:35]: Gotcha - that makes sense!

**Pull request merged**